po/es.po

```diff
@@ -15260,39 +15262,39 @@ msgstr "No hay ninguna cámara con soporte de captura disponible para su uso..."
 
 #: ../src/views/view.c:2143
 msgid "Left click"
-msgstr "Clic izquierdo"
+msgstr "Clic"
 
 #: ../src/views/view.c:2146
 msgid "Right click"
-msgstr "Clic derecho"
+msgstr "Clic Secundario"
 
 #: ../src/views/view.c:2149
 msgid "Middle click"
-msgstr "Clic medio"
+msgstr "Clic Medio"
 
 #: ../src/views/view.c:2152
 msgid "Scroll"
 msgstr "Desplazamiento"
 
 #: ../src/views/view.c:2155
 msgid "Left double-click"
-msgstr "Doble clic izquierdo"
+msgstr "Pulsación Doble"
 
 #: ../src/views/view.c:2158
 msgid "Right double-click"
-msgstr "Doble clic derecho"
+msgstr "Doble Clic Secundario"
```

Comment:
Case is not right at all "Doble clic secundario".

Reply:
As in the following explanation...